.gitignore

```diff
@@ -5,4 +5,5 @@ obj/
 launchSettings.json
 artifacts/
 msbuild.binlog
-.vscode/
\ No newline at end of file
+.vscode/
+src/Microsoft.AspNetCore.Blazor.Browser.JS/package-lock.json
```

Comment:
Could you revert the changes to this file? We don't want to exclude `package-lock.json` from source control.